tikv/raftstore/worker.go

```diff
@@ -446,6 +446,7 @@ func (r *regionTaskHandler) handle(t task) {
 	regionTask := t.data.(*regionTask)
 	switch t.tp {
 	case taskTypeRegionApply:
+		metrics.ApplyTaskWaitTimeHistogram.Observe(time.Now().Sub(regionTask.startTime).Seconds())
```

Comment:
use `time.Since(regionTask.startTime)` is simpler.

Reply:
Updated.